### PR TITLE
CHE-1108 remove creating ExternalFolderChange instance

### PR DESCRIPTION
**Pull request by evidolob**

### CODE REVIEW

File: plugins/plugin-java/che-plugin-java-ext-jdt/org-eclipse-jdt-ui/src/main/java/org/eclipse/jdt/internal/core/ChangeClasspathOperation.java [hunk 1]
```diff
@@ -58,7 +58,7 @@ protected void classpathChanged(ClasspathChange change, boolean refreshExternalF
 			new ProjectReferenceChange(project, change.oldResolvedClasspath).updateProjectReferencesIfNecessary();
 //
 			// and ensure that external folders are updated as well
-			new ExternalFolderChange(project, change.oldResolvedClasspath).updateExternalFoldersIfNecessary(refreshExternalFolder, null);
+			//new ExternalFolderChange(project, change.oldResolvedClasspath).updateExternalFoldersIfNecessary(refreshExternalFolder, null);
```
Comment (skabashnyuk) [inline plugins/plugin-java/che-plugin-java-ext-jdt/org-eclipse-jdt-ui/src/main/java/org/eclipse/jdt/internal/core/ChangeClasspathOperation.java:62]: Remove please if this code is no longer needed. Or explain why it's commented.

Comment (evidolob) [inline plugins/plugin-java/che-plugin-java-ext-jdt/org-eclipse-jdt-ui/src/main/java/org/eclipse/jdt/internal/core/ChangeClasspathOperation.java:62]: It's Eclipse code, I prefer to keep it for future update

Comment (vparfonov) [inline plugins/plugin-java/che-plugin-java-ext-jdt/org-eclipse-jdt-ui/src/main/java/org/eclipse/jdt/internal/core/ChangeClasspathOperation.java:62]: +1

**Comment by vparfonov** [comment]
ok

**Pull request merged**